alacritty/src/config/color.rs

```diff
@@ -18,15 +18,26 @@ pub struct Colors {
     pub line_indicator: LineIndicatorColors,
     pub hints: HintColors,
     pub transparent_background_colors: bool,
+    pub information_bar: BarColors,
 }
 
 impl Colors {
-    pub fn search_bar_foreground(&self) -> Rgb {
-        self.search.bar.foreground.unwrap_or(self.primary.background)
+    pub fn information_bar_foreground(&self) -> Rgb {
+        // Legacy field first.
```

Comment:
The comment isn't really necessary, that should be a familiar thing within the codebase so no need to point it out every time.